laboratory/views.py

```
@@ -1,8 +1,26 @@
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.exceptions import PermissionDenied
 from .models import LabResult
 from .serializers import LabResultSerializer
 
 
 class LabResultViewSet(viewsets.ModelViewSet):
     queryset = LabResult.objects.all().order_by('-taken_at')
-    serializer_class = LabResultSerializer
\ No newline at end of file
+    serializer_class = LabResultSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)